server/api/views/equipment.py

```diff
@@ -37,6 +46,64 @@ class EquipmentItemViewSet(viewsets.ModelViewSet):
     serializer_class = EquipmentItemSerializer
 
 
-def list_equipment(request, format=None):
-    all_equipment = list(EquipmentType.objects.values())
-    return JsonResponse(all_equipment, safe=False)
\ No newline at end of file
+class EquipmentRequestViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows Equipment Requests to be viewed or edited.
+    """
+    queryset = EquipmentRequest.objects.all()
+    serializer_class = EquipmentRequestSerializer
+
+def get_availability(request, request_out, request_in, equipment_item_id):
+    request_out_fmt = datetime.datetime.strptime(request_out, "%Y-%m-%dT%H:%M:%S%z")
+    request_in_fmt = datetime.datetime.strptime(request_in, "%Y-%m-%dT%H:%M:%S%z")
```

Comment:
Have you considered timezones?

Reply:
The %z directive should take into account the timezone of the request.